### DISCUSSION TITLE
Importing flopy modfies PATH environment variable and overrides user settings

### DISCUSSION THREAD

**Discussion by dbrakenhoff** [modflowpy/flopy#2032] [Q&A] [answered]
I'm posting this here, as I'm not sure whether this qualifies as bug or turns out to be more of a user question.

I was working with a custom version of Modflow6 in a separate environment. In my shell, `which mf6` pointed nicely to my custom mf6 binary. After starting up Python and importing flopy, `shutil.which("mf6")` then pointed to the binaries I downloaded with the get-modflow utility, and not my custom one. This screenshot shows what I mean:

![image](https://github.com/modflowpy/flopy/assets/6552225/5a20df15-ddb1-41c1-9cee-5c1faaf350ff)

That is a bit annoying, because that then requires you to either edit your PATH again, or explicitly tell your MFSimulations which binary they should use. 

So, is this desired behavior? Or would it be better to append to the PATH instead of prepending flopy's default bin folder? 

The relevant code in `mbase.py`:

https://github.com/modflowpy/flopy/blob/9c6c2343373261c989a223a6390d2718f4ceb3f7/flopy/mbase.py#L31-L36

Thanks in advance for taking a look!

**Comment by wpbonelli** [comment]
Appending instead of prepending to PATH in `mbase.py` makes sense to me. @mwtoews and others may have thoughts

**Comment by mwtoews** [comment, marked as answer]
This analysis make sense to append the path instead of prepending, so agree that it could change for a future release.

Another workaround is to remove the flopy bin folder.

**Comment by wpbonelli** [comment]
addressed by #2036, #2037
